ResumableUpload.CS.Sample/README.html

```diff
@@ -0,0 +1,42 @@
+<html>  
+<title>Google .NET Client API &ndash; ResumableUpload.CS.Sample</title>  
+<body>  
+<h2>Instructions for the Google .NET Client API &ndash; ResumableUpload.CS.Sample</h2>  
+  
+<h3>Browse Online</h3>  
+  
+<ul>  
+  <li><a href="http://code.google.com/p/google-api-dotnet-client/source/browse/?repo=samples#hg%2FResumableUpload.CS.Sample">Browse Source</a>,  
+   or main file <a href="http://code.google.com/p/google-api-dotnet-client/source/browse/ResumableUpload.CS.Sample/ResumableUpload.cs?repo=samples">ResumableUpload.cs</a></li>  
+</ul>  
+   
+<h3>1. Checkout Instructions</h3>  
+ 
+<p><b>Prerequisites:</b> Install Visual Studio, and <a href="http://mercurial.selenic.com/">Mercurial</a>.</p>  
+  
+<pre><code>cd <i>[someDirectory]</i>  
+hg clone https://code.google.com/p/google-api-dotnet-client.samples/ google-api-dotnet-client-samples</code></pre>  
```

Comment:
Alas, Google Code has been shut down. So this should be updated to point to GitHub. But if we are changing the contents of this file, here are some more meaningful suggestions:

- Would you be opposed to converting it to Markdown? Then when viewed from the GitHub UI it would be rendered correctly.
- Eliminate most of the boiler plate? If you are reading the source code, you probably already have your VCS client set up. The "API access instructions" are crucial however.